shelley-ma/impl/src/Cardano/Ledger/ShelleyMA/Rules/Utxo.hs

```diff
@@ -250,6 +252,18 @@ utxoTransition = do
           outputs
   null outputsTooSmall ?! OutputTooSmallUTxO outputsTooSmall
 
+  let outputsTooBig =
+        filter
+          ( \out ->
+              let v = getField @"value" out
+               in Val.size v > 10000
```

Comment:
Since this also applies in Allegra, we need to validate against the current chain

Reply:
yes indeed!

for `Coin`, `size` is always 1, but of course we should still validate.

https://github.com/input-output-hk/cardano-ledger-specs/blob/721d350cbd71dae2ddd0a6a1b57901cc1940856b/shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Val.hs#L96

Reply:
Oh, that's a good point! We're probably fine then :-)